### PR TITLE
(chore) Pin versions of all dependencies to their current minor release.

### PR DESCRIPTION
**Pull request by reem**

### CODE REVIEW

**Comment by reem** [comment]
cc @sfackler no more *s!

**Comment by sfackler** [comment]
:confetti_ball:

File: Cargo.toml [hunk 1]
```diff
@@ -24,15 +24,15 @@ default = ["ssl"]
...
-plugin = "*"
-modifier = "*"
-error = "*"
-log = "*"
-conduit-mime-types = "*"
-lazy_static = "*"
-num_cpus = "*"
+typemap = "0.3"
+url = "0.2"
+plugin = "0.2*"
```
Comment (sfackler) [inline Cargo.toml]: Looks like a trailing wildcard here

Comment (reem) [inline Cargo.toml]: Oh woops, what behavior does this actually have?

Comment (sfackler) [inline Cargo.toml]: No idea. Seems like it shouldn't parse.

Comment (reem) [inline Cargo.toml]: Surprisingly enough it does work...

**Pull request merged**